packaging/windows/OpenRA.nsi

```diff
@@ -126,35 +126,41 @@ Section "Game" GAME
 	File /r "${SRCDIR}\mods\modcontent"
 
 	SetOutPath "$INSTDIR"
-	File "${SRCDIR}\RedAlert.exe"
-	File "${SRCDIR}\TiberianDawn.exe"
-	File "${SRCDIR}\Dune2000.exe"
-	File "${SRCDIR}\OpenRA.Game.exe"
-	File "${SRCDIR}\OpenRA.Game.exe.config"
-	File "${SRCDIR}\OpenRA.Utility.exe"
-	File "${SRCDIR}\OpenRA.Server.exe"
-	File "${SRCDIR}\OpenRA.Platforms.Default.dll"
-	File "${SRCDIR}\ICSharpCode.SharpZipLib.dll"
-	File "${SRCDIR}\FuzzyLogicLibrary.dll"
-	File "${SRCDIR}\Open.Nat.dll"
+	File "${SRCDIR}\*.exe"
+	File "${SRCDIR}\*.exe.config"
+	File "${SRCDIR}\*.dll"
```

Comment:
The launchers refuse to run due to a missing `WindowsLauncher.dll`. This is confusing, because this file doesn't appear to even exist?

Reply:
The runtime also refuses to run unless we also package `SOS_README.md`, which is a real WTF.